Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/css/main.css

```diff
@@ -740,8 +740,8 @@ span.dnnFormError:after {
     width: 80px;
     height: 103px;
     background: var(--dnn-pb-menu-brand-background);
-    -ms-background-size: 32px auto;
-    background-size: 32px auto;
+    -ms-background-size: var(--dnn-pb-menu-brand-background-size);
+    background-size: var(--dnn-pb-menu-brand-background-size); /* 32px auto; */

```

Comment:
I would either remove the comment (it would lie if someone changes it and be just confusing) or make it a fallback (so that it defaults to that size if not specified):

var(--dnn-pb-menu-brand-background-size, 32px auto);

Reply:
I'll remove the comment. Not needed since the default (original) values `32px auto` is in theme.css and there is no case where a fallback is needed.